MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,747 @@
+---
+layout: post
+title: Floating label layout in .NET MAUI DataForm control | Syncfusion
+description: Learn about the Floating label layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Floating label Layout in .NET MAUI DataForm (SfDataForm)
+
+## Overview
+
+The floating label layout is a powerful feature that enhances the user experience and makes it easier for users to interact with forms.The dataform supports floating label layout which includes assistive labels, leading and trailing icons, and a password toggle icon to show or hide a password. It offers three different kinds of containers, including filled, outlined, and none. Setting "LayoutType" to "TextInputLayout" enables floating label layout.
+
+## Layout type
```

Comment:
Changing the layout type for the data form

Reply:
Changed heading